docs/en/developers/03_Backend/06_Reverse_Proxy_Setup.md

```diff
@@ -0,0 +1,81 @@
+## Using Caddy as a Reverse Proxy with a Subfolder
+
+To set up FreshRSS behind a reverse proxy with Caddy and using a subfolder, follow these steps:
+
+1. **Configure Caddyfile:**
+
+    Update your Caddyfile with the following configuration:
+
+    ```plaintext
+    yourdomain.com {
+        redir /freshrss /freshrss/i/
+        route /freshrss* {
+            uri strip_prefix /freshrss
+            reverse_proxy freshrss:80 {
+                header_up Host {host}
+                header_up X-Real-IP {remote}
+                header_up X-Forwarded-Proto {scheme}
+                header_up X-Forwarded-Host {host}
+                header_up X-Forwarded-For {remote}
+                header_up X-Forwarded-Ssl {on}
+                header_up X-Forwarded-Prefix "/freshrss/"
+            }
+        }
+    }
+    ```
+
+    Replace `yourdomain.com` with your actual domain and `freshrss` with the subfolder where FreshRSS is hosted.
+
+2. **Update FreshRSS Configuration:**
+
+    Open the `config.php` file in your FreshRSS installation and update the `base_url` parameter to match the subfolder configuration:
+
+    ```php
+    'base_url' => 'https://yourdomain.com/freshrss',
+    ```
+
+    Replace `yourdomain.com` with your actual domain and `freshrss` with the subfolder name specified in the Caddyfile.
+
+3. **Restart Caddy and FreshRSS:**
+
+    Restart Caddy to apply the configuration changes:
+
+    ```bash
+    systemctl restart caddy
+    ```
+
+    Restart FreshRSS to ensure that it recognizes the new base URL:
+
+    ```bash
+    docker-compose restart freshrss
+    ```
+
+4. **Access FreshRSS:**
+
+    FreshRSS should now be accessible at `https://yourdomain.com/freshrss`.
+
+### Example Caddyfile Entry
+
+```plaintext
+yourdomain.com {
+    redir /freshrss /freshrss/i/
+    route /freshrss* {
+        uri strip_prefix /freshrss
+        reverse_proxy freshrss:80 {
+            header_up Host {host}
+            header_up X-Real-IP {remote}
+            header_up X-Forwarded-Proto {scheme}
+            header_up X-Forwarded-Host {host}
+            header_up X-Forwarded-For {remote}
+            header_up X-Forwarded-Ssl {on}
+            header_up X-Forwarded-Prefix "/freshrss/"
+        }
+    }
+}
+```
+
+Replace `yourdomain.com` with your actual domain and `freshrss` with the subfolder name where FreshRSS is hosted.
```

Comment:
```suggestion
Replace `example.net` with your actual domain and `freshrss` with the subfolder name where FreshRSS is hosted.
```

Please use one of the domains reserved for examples / documentations, so we do not start linking to random existing sites: https://en.wikipedia.org/wiki/Example.com